.prettierrc.js

```diff
@@ -1,7 +1,7 @@
-module.exports =  {
-    semi:  true,
-    trailingComma:  'none',
-    singleQuote:  false,
-    printWidth:  120,
-    tabWidth:  4
+module.exports = {
+    semi: true,
+    trailingComma: 'none',
+    singleQuote: false,
+    printWidth: 160,
```

Comment:
I think in future we should do formatting changes in a separate PR. It made it quite tough for me to see some of the actual changes on this PR, especially in the tests

Reply:
Sorry about that; I disabled the formatOnSave option for prettier.